### PR TITLE
Photo thumbnails

### PR DESCRIPTION
**Pull request by rsvargas**
photo thumbnails silver feather

Was this what you wanted?

### CODE REVIEW

File: css/base.css [hunk 1]
```diff
@@ -255,12 +255,19 @@ input[type='text'] {
 .tweet .text {
   clear: right;
   word-wrap: break-word;
+  white-space: pre-wrap;
```
Comment (kav2k) [inline css/base.css]: Why was that needed?

Comment (rsvargas) [inline css/base.css]: It was supposed to solve a problem that does not exist anymore. Multi-line retweets that span beyond the user's avatar were not aligning correctly. But I wasn't able to reproduce it. It is not needed anymore as it was solved somewhere else (altough I don't know where)

Comment (kav2k) [inline css/base.css]: Might also be needed for your `<br>` padding of the picture; I'm going to change that to 0.5em margins.

**Comment by kav2k** [comment]
This currently fails to show several pictures from the same tweet.

**Comment by rsvargas** [comment]
and also fails to show animated gifs

**Comment by kav2k** [comment]
I think the fact that it fails to show several images is severe: it misleads users, making them think there's only one image attached. I'll look into it.

**Comment by kav2k** [comment]
Closing: merged into branch image-previews

**Comment by kav2k** [comment]
Cross-reference note: all relevant to #15
